packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/utils.js

```diff
@@ -132,6 +134,20 @@ describe('plugin-meetings', () => {
         });
       });
     });
+
+    describe('joinMeeting', () => {
+      it('#Should call `meetingRequest.joinMeeting', async () => {
```

Comment:
would be nice to have a test that also checks that we fall back to the sipUri if meetingJoinUrl is missing

Reply:
done****